OP2Helper.h

```diff
@@ -52,9 +52,6 @@ bool operator!= (const LOCATION& loc1, const LOCATION &loc2);
 // Starting Resources
 void InitPlayerResources(int playerNum, const ResourceSet& resourceSet = CES1ResourceSet);
 
-// Constructing Tubes/Walls
-void CreateTubeOrWallLine(int x1, int y1, int x2, int y2, map_id type);

```

Comment:
Rather than removing public access, and breaking old code, I was thinking maybe it should be marked `[[deprecated]]` instead.

I see you've later changed the parameters from unpacked `int`s to `LOCATION` structs, so that would break old code. Not sure if I want to force people to pack parameters into `LOCATION` structs, though I certainly don't object to allowing it as a convenience.